client/src/common/enums/app/env.enum.js

```diff
@@ -1,8 +1,8 @@
-const { REACT_APP_SOCKET_SERVER, REACT_APP_API_PATH } = process.env;
+const { VITE_SOCKET_SERVER, VITE_API_PATH } = import.meta.env;
```

Comment:
Not sure that destructurisation will work properly, I had an issue with it on another project